src/main/java/se/sundsvall/disturbance/api/model/Disturbance.java

```diff
@@ -1,14 +1,19 @@
 package se.sundsvall.disturbance.api.model;
 
+import static io.swagger.v3.oas.annotations.media.Schema.RequiredMode.REQUIRED;
+
 import java.time.OffsetDateTime;
 import java.util.List;
 import java.util.Objects;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 
+import se.sundsvall.disturbance.api.validation.OneOf;
+
 import io.swagger.v3.oas.annotations.media.ArraySchema;
 import io.swagger.v3.oas.annotations.media.Schema;
 import io.swagger.v3.oas.annotations.media.Schema.AccessMode;
+import jakarta.validation.constraints.NotNull;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Don't think this is needed, since dept44 includes the property `spring.jackson.default-property-inclusion=NON_NULL`